app/discovery/provider/docker.go

```diff
@@ -124,7 +114,7 @@ func (d *Docker) List() ([]discovery.URLMapper, error) {
 
 		srcRegex, err := regexp.Compile(srcURL)
 		if err != nil {
-			return nil, fmt.Errorf("invalid src regex %s: %w", srcURL, err)
+			return nil, errors.Wrapf(err, "invalid src regex %s", srcURL)
```

Comment:
one of prev PR changes pkg/errors for stdlib everywhere, we want to keep it like this everywhere

Reply:
Missed that one while rebasing. Fixed.